UI/win-update/updater/updater.cpp

```diff
@@ -606,20 +606,13 @@ static inline bool FileExists(const wchar_t *path)
 
 static bool NonCorePackageInstalled(const char *name)
 {
-	if (is32bit) {
-		if (strcmp(name, "obs-browser") == 0) {
+	if (strcmp(name, "obs-browser") == 0) {
+		if (is32bit)
```

Comment:
tbh we can drop all 32 bit checks in the updater now that we've discontinued 32 bit OBS.

Reply:
Had the same thought, though I think it's better to have that in a separate PR and commit.